doc/sphinx/changes.rst

```diff
@@ -654,6 +654,14 @@ Kernel
   (`#18599 <https://github.com/coq/coq/pull/18599>`_,
   by Guillaume Melquiond).
 
+Notations
+^^^^^^^^^
+
+- **Changed:**
+  :warn:`Found no matching notation to enable or disable.` is a warning instead of an error
+  (`#18670 <https://github.com/coq/coq/pull/18670>`_,
+  by Pierre Roux).
```

Comment:
cc @proux01

Reply:
I could maybe have written "is now" rather than "is", but that's fine.